wazimap_ng/boundaries/management/commands/rebuildshpfiles.py

```diff
@@ -0,0 +1,104 @@
+import os
+from hmac import new
+from typing import Any, Dict, List
+
+import shapefile
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """
+    This command reads shapefiles in a specific path and rebuilds them while splitting them
+    into smaller files based on the level so that they can correcly be loaded with the appropriate
+    level into the geography heirarchy.
+
+    If the source directory is
+    - ./Country/country.{shp,shx,dbf}
+
+    and the shapefile contains different levels, provice and municipality, the output will be:
+    - ./Country/Province/province.{shp,shx,dbf}
+    - ./Country/Municipality/municipality.{shp,shx,dbf}
```

Comment:
And wazimap **_needs_** and **_can_** handle this nested directory structure?

Reply:
Hello @kilemensi, no it doesn't **_need_** the _**nested**_ directory structure for the output, but it's super helpful if you have your files grouped into directories for input. I've updated it in the other PR to specify an output directory. So what's happening is actually:

Input: 
```
Country/
└── country.shp
```

Output: 
```
Country/
└── <output-dir>/
    ├── City.shp
    ├── Town.shp
    └── District.shp
```

For simplicity I'm only showing the `.shp` file. But each shape file comes with `.dbf` and `shx` files to complete the set. They are always in threes and so the chances of you having them in a directory in the first place is very high.

Reply:
@kilemensi the `loadshp` command loads file by file. The other PR I've worked on can now take advantage of this split directory and load all the files in one go by calling the `loadshp` command with appropriate arguments for each shape file. It takes a glob pattern that matches all the `.shp` files you intend to load.